src/lib.rs

```diff
@@ -139,6 +139,9 @@ pub fn tidy_list(req: TidyRequest) -> Vec<String> {
 
         new_word = new_word.trim_start().trim_end().to_string();
```

Comment:
Sorry for the drive by review, feel free to ignore: I think you could avoid the intermediate allocation here since [the unicode_normalization::UnicodeNormalization trait](https://docs.rs/unicode-normalization/latest/unicode_normalization/trait.UnicodeNormalization.html#impl-UnicodeNormalization%3CChars%3C%27a%3E%3E-for-%26%27a%20str) is implemented for iterators that yield chars. Something like:

```rust
new_word = new_word.trim().nfc().collect::<String>()
```

Reply:
Yep, was thinking the same.